### PR TITLE
Endpoint to retrieve thread recipients

### PR DESCRIPTION
**Pull request by pedrocarloto**

### CODE REVIEW

File: apps/innovations/_services/innovations.service.ts [hunk 1]
```diff
@@ -69,6 +69,8 @@ import { groupBy, isString, mapValues, omit, pick, snakeCase } from 'lodash';
 import { BaseService } from './base.service';
 import type { InnovationDocumentService } from './innovation-document.service';
 import SYMBOLS from './symbols';
+import { InnovationRelevantOrganisationsStatusView } from '@innovations/shared/entities/views/innovation_relevant_organisations_status_view.entity';
+import type { InnovationRelevantOrganisationsStatusEnum } from 'libs/shared/enums';
```
Comment (diogo-costa-bjss) [inline apps/innovations/_services/innovations.service.ts:73]: Wrong import

File: apps/innovations/_services/innovations.service.ts [hunk 2]
```diff
@@ -69,6 +69,8 @@ import { groupBy, isString, mapValues, omit, pick, snakeCase } from 'lodash';
 import { BaseService } from './base.service';
 import type { InnovationDocumentService } from './innovation-document.service';
 import SYMBOLS from './symbols';
+import { InnovationRelevantOrganisationsStatusView } from '@innovations/shared/entities/views/innovation_relevant_organisations_status_view.entity';
```
Comment (diogo-costa-bjss) [inline apps/innovations/_services/innovations.service.ts:72]: `@innovations/shared/entities`

File: apps/innovations/_services/innovations.service.ts (hunk 1 above)
Comment (pedrocarloto) [reply apps/innovations/_services/innovations.service.ts:73]: Changed

File: apps/innovations/_services/innovations.service.ts (hunk 2 above)
Comment (pedrocarloto) [reply apps/innovations/_services/innovations.service.ts:72]: How do I change this on?
image
In the same file we also have:
`import {
  InnovationListViewWithoutNull,
  InnovationProgressView
} from '@innovations/shared/entities/views/innovation-progress.view.entity';`

**Review by eduarda-meirinhos-bjss — APPROVED** [review]

**Pull request merged**